flycheck.el

```diff
@@ -6683,22 +6683,15 @@ shell execution."
   (let* ((args (flycheck--checker-substituted-shell-command-arguments checker))
          (program
           (or (flycheck-find-checker-executable checker)
-              (user-error "Cannot find `%s' using `flycheck-executable-find'"
+              (user-error "Cannot find `%s' using `flycheck-find-checker-executable'"
                           (flycheck-checker-executable checker))))
          (wrapped (flycheck--wrap-command program args))
-         (abs-prog
-          ;; The executable path returned by `flycheck-command-wrapper-function'
-          ;; may not be absolute, so expand it here.  See URL
-          ;; `https://github.com/flycheck/flycheck/issues/1461'.
-          (or (executable-find (car wrapped))
-              (user-error "Cannot find `%s' using `executable-find'"
-                          (car wrapped))))
          (command (mapconcat #'shell-quote-argument
-                             (cons abs-prog (cdr wrapped)) " ")))
+                             (cons program (cdr wrapped)) " ")))
```

Comment:
This appears to be obsolete. `flycheck-find-checker-executable` calls `flycheck-executable-find` which calls `executable-find` which returns an absolute path.